From c135a57d447f86971928777e776d4ea43d6a5c96 Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Thu, 6 Dec 2012 09:41:45 -0800
Subject: [PATCH 050/196] HBASE-7069 HTable.batch does not have to be synchronized

Reason: Bug
Author: Lars Hofhansl
Ref: CDH-9159

git-svn-id: https://svn.apache.org/repos/asf/hbase/branches/0.94@1403808 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 2e18c3ddc186a38a464be79ca4af335d7692f7db)
---
 .../org/apache/hadoop/hbase/client/HTable.java     |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/main/java/org/apache/hadoop/hbase/client/HTable.java b/src/main/java/org/apache/hadoop/hbase/client/HTable.java
index 9bea85f..bc5f6e3 100644
--- a/src/main/java/org/apache/hadoop/hbase/client/HTable.java
+++ b/src/main/java/org/apache/hadoop/hbase/client/HTable.java
@@ -688,7 +688,7 @@ public class HTable implements HTableInterface {
    * {@inheritDoc}
    */
   @Override
-  public synchronized void batch(final List<?extends Row> actions, final Object[] results)
+  public void batch(final List<?extends Row> actions, final Object[] results)
       throws InterruptedException, IOException {
     connection.processBatch(actions, tableName, pool, results);
   }
@@ -697,7 +697,7 @@ public class HTable implements HTableInterface {
    * {@inheritDoc}
    */
   @Override
-  public synchronized Object[] batch(final List<? extends Row> actions) throws InterruptedException, IOException {
+  public Object[] batch(final List<? extends Row> actions) throws InterruptedException, IOException {
     Object[] results = new Object[actions.size()];
     connection.processBatch(actions, tableName, pool, results);
     return results;
-- 
1.7.0.4

